Automattic/wp-calypso
# Rename and move `plans-v2` folder

### Changes proposed in this Pull Request

As part of a cleanup process initiated by #47466 and #47474, this PR renames and moves the folder `client/my-sites/plans-v2` to `client/my-sites/plans/jetpack-plans`. The `v2` suffix was handy to quickly implement the offer reset, but would most likely be confusing in the future.

### Implementation notes

- Moved content of `client/my-sites/plans-v2` to `client/my-sites/plans/jetpack-plans`
- Renamed class names matching `plans-v2__` to `jetpack-plans__`
- Renamed routing function `plansV2` to `jetpackPlans`

### Testing instructions

- Download the PR
- Run both Calypso and Jetpack cloud concurrently
- Visit the plans page in both platforms
- Check that it looks like production and that you see no error in the browser console
